### DISCUSSION TITLE
Count of elements in a collection for a property on an event

### DISCUSSION THREAD

**Discussion by c0shea** [datalust/seq-tickets#1579] [Help] [answered]
My API is logging the requests to Seq with the full request object in a `RequestDto` property. Some of the request objects are collections containing one or more objects of the same type.

Is it possible to count the number of elements in the collection? In this example, the count is 1. I'm trying to quickly see how many elements are in the collection across requests to identify any outliers.

```json
[{"__type":"Api.Model.Core.FindVendorItems, Api.Model","VendorIds":[7500],"OrderCodes":[937256]}]

**Comment by liammclennan** [comment, marked as answer]
Hi Connor,

Perhaps something like `select Length(RequestDto.OrderCodes) from stream`?

https://docs.datalust.co/docs/scalar-functions#lengtharg

**Reply by c0shea** [reply]
Thanks @liammclennan! I hadn't realized that Length would operate on arrays. The docs made it seem like it was only good for measuring string length.
